src/applications/disability-benefits/781/pages/ptsdSecondaryLocation.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { PtsdNameTitle } from '../helpers';
+
+const ptsdLocationDescription = () => (
+  <div>
+    <h5>Event location</h5>
+    <p>
+      Where did the event happen? Please be as specific as you can and include
+      the name of the city, state, country, province, landmark, or military
+      installation.
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': PtsdNameTitle,
```

Comment:
Does this need to be updated to pass in the formType?

Reply:
@robfelder , short answer is yes, long answer, I'm currently refactoring Location page for another story, I got ahead of myself in this branch, and added the code for that story.

Reply:
gotcha